app/src/main/java/org/oppia/app/topic/revisioncard/ReturnToTopicClickListener.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.app.topic.revisioncard
+
+/** Listener interface for when  Return to Topic is clicked in the UI. */
```

Comment:
Nit update: /** Listener to route to [TopicActivity] when clicked on Return to Topic button. */

Reply:
Done